include/deal.II/base/geometry_info.h

```diff
@@ -1211,7 +1211,7 @@ struct GeometryInfo<0>
    * using range-based for loops of the following kind:
    * @code
    *   for (auto &cell : triangulation.active_cell_iterators())
-   *     for (auto face_index : GeometryInfo<dim>::face_indices)
+   *     for (auto face_index : GeometryInfo<dim>::face_indices())
```

Comment:
Turns out that I forgot the parentheses in my first attempt at this patch (compiler didn't like that) and had to do a global search-replace -- which uncovered that I had made the same mistake writing the documentation in #9415...